templates/candlepin.conf.erb

```diff
@@ -34,6 +34,9 @@ candlepin.crl.file=<%= scope['candlepin::crl_file'] %>
 candlepin.ca_key_password=<%= scope['candlepin::ca_key_password'] %>
 <%- end -%>
 
+# Default is every hour - override to once every 24h
```

Comment:
Nit: technically it's a parameter and the `override to once every 24h` isn't guaranteed to be accurate, but for most users it'll be true and the others probably won't care about the comment.